components/Navbar.vue

```diff
@@ -1,43 +1,50 @@
 <template>
-  <header class="navbar">
-    <nav>
-      <div class="box">
-        <div class="logo">
-          <a href="https://realdevsquad.com/">
-            <img
-              src="~/assets/Real-Dev-Squad@1x.png"
-              alt="Real Dev Squad logo"
-            />
-          </a>
-        </div>
-        <div class="hamburger-menu" @click="toggleClicked(isClicked)">
-          <span></span>
-          <span></span>
-          <span></span>
-        </div>
-      </div>
-      <ul :class="[isClicked ? 'show-menu' : 'hide-menu']">
-        <li>
-          <a href="https://welcome.realdevsquad.com/">Welcome</a>
-        </li>
-        <li>
-          <a href="https://www.realdevsquad.com/events.html">Events</a>
-        </li>
-        <li>
-          <a href="https://members.realdevsquad.com/">Members</a>
-        </li>
-        <li>
-          <a href="https://crypto.realdevsquad.com/">Crypto</a>
-        </li>
-        <li>
-          <a href="https://status.realdevsquad.com/">Status</a>
-        </li>
-        <li>
-          <a class="active" href="https://goals.realdevsquad.com/">Goals</a>
-        </li>
-      </ul>
-    </nav>
-  </header>
+  <nav class="navbar">
+    <button class="hamburger" @click="toggleClicked(isClicked)">
+      <div class="hamburger__bar" />
+      <div class="hamburger__bar" />
+      <div class="hamburger__bar" />
```

Comment:
We can use the here maybe or if we are not using the `menu.png`, can we remove it ?

Reply:
The png isn't that high quality, that's why avoided it.

Reply:
Got it